### PR TITLE
Merge 19.3 code freeze

### PR DESCRIPTION
**Pull request by mokagio**
BTW: I'll followup this with a PR to merge 19.2.1 into 19.3 soon.

### CODE REVIEW

**Review by mokagio — COMMENTED** [review]
- [x] Switch to production Pods
- [x] New version header in `RELEASE-NOTE.txt` 
- [x] `Localizable.strings` updated 
- [x] `release_notes.txt` updated with notes from `RELEASE-NOTE.txt` for current version 
- [x] Version update in `.xcconfig`, `Deliverfile` (not for WP), and `download_metadata.swift`

File: WordPress/Jetpack/Resources/release_notes.txt [hunk 1]
```diff
@@ -1,6 +1,4 @@
 * [*] Site previews: Reduced visual flickering when previewing sites and templates. [#17861]
 * [*] Stats: Scroll to new Insights card when added. [#17894]
 * [*] Add "Copy Link" functionality to Posts List and Pages List [#17911]
-* [*] [Jetpack-only] Enables the ability to use and create WordPress.com sites, and enables the Reader tab. [#17914, #17948]
```
Comment (AliSoftware) [inline WordPress/Jetpack/Resources/release_notes.txt]: This should NOT have been deleted from the Jetpack Release Notes. On the contrary, that's one key entry that should be in the Jetpack notes but not on the WP ones

File: WordPress/Resources/release_notes.txt [hunk 2]
```diff
@@ -1,6 +1,5 @@
 * [*] Site previews: Reduced visual flickering when previewing sites and templates. [#17861]
 * [*] Stats: Scroll to new Insights card when added. [#17894]
 * [*] Add "Copy Link" functionality to Posts List and Pages List [#17911]
-* [*] [Jetpack-only] Enables the ability to use and create WordPress.com sites, and enables the Reader tab. [#17914, #17948]
+* [*] Enables the ability to use and create WordPress.com sites, and enables the Reader tab. [#17914, #17948]
```
Comment (AliSoftware) [inline WordPress/Resources/release_notes.txt]: This should be on the Jetpack release notes, but NOT on the WordPress release notes

**Review by AliSoftware — DISMISSED** [review]
I think you mixed up the WordPress vs Jetpack release notes, as this time around the entry that needs deleting is the one from WordPress, given it's a JP-only one.

[EDIT] Oh sorry you fixed that already in https://github.com/wordpress-mobile/WordPress-iOS/pull/17997/commits/7cca0fa26d81f02abf968b4860b1b11728b5e055, nvm 😅

File: WordPress/Jetpack/Resources/release_notes.txt [hunk 3]
```diff
@@ -1,9 +1,5 @@
...
-- Text blocks in the block editor now adjust along with the text as you change the font size and line height for block themes.
-- We redesigned some of the screens in the app, including My Site. Looking fresh!
-- We love a clean inbox, so we added a button that lets you mark all filtered notifications as read.
-- In the site tab’s Configure > People menu, we added an email filter that allows you to see, learn more about, and even remove email followers.
-- In threaded comments, we removed the Share icon and instead added the ability to moderate comments within a thread.
-- If you change the media in an Image block that’s also used as a post’s Featured Image, you’ll see a prompt asking if you’d like to replace the Featured Image while you’re at it.
-- We improved the Stats navigation so you can move between Days and Insights more easily. You can also create new stat cards directly from the Stats screen using our new navigation icon.
+* [*] Site previews: Reduced visual flickering when previewing sites and templates. [#17861]
+* [*] Stats: Scroll to new Insights card when added. [#17894]
+* [*] Add "Copy Link" functionality to Posts List and Pages List [#17911]
```
Comment (AliSoftware) [inline WordPress/Jetpack/Resources/release_notes.txt:3]: External contribution 🎉

File: WordPress/Jetpack/Resources/release_notes.txt [hunk 4]
```diff
@@ -1,9 +1,5 @@
...
+* [*] Enables the ability to use and create WordPress.com sites, and enables the Reader tab. [#17914, #17948]
```
Comment (AliSoftware) [inline WordPress/Jetpack/Resources/release_notes.txt]: IMHO this should be a `[**]` if not a `[***]` (and thus top of the list in the final notes) as this is a major set of new features for the Jetpack app. wdyt @ashiagr @leandroalonso ?

Comment (leandroalonso) [reply WordPress/Jetpack/Resources/release_notes.txt]: Agree!

Comment (mokagio) [reply WordPress/Jetpack/Resources/release_notes.txt]: Fair point.

```suggestion
* [***] Enables the ability to use and create WordPress.com sites, and enables the Reader tab. [#17914, #17948]
```

File: WordPress/Jetpack/Resources/release_notes.txt (hunk 3 above)
Comment (mokagio) [reply WordPress/Jetpack/Resources/release_notes.txt:3]: Yep. I had a reminder set back when that went in to ping them, just waiting for this to merge ⌛

**Review by AliSoftware — APPROVED** [review]
✅  Stable version of pods
✅  New section in RELEASE-NOTES.txt
✅  Changes from hotfix 19.2.1 branch (via #17998)
✅  New strings frozen
✅  WP & JP release notes extracted and audited
✅ Version bump in xcconfig

**Pull request merged**